app/src/components/DemoButton/DemoButton.test.tsx

```diff
@@ -4,21 +4,21 @@ import { DemoButton } from './DemoButton'
 
 describe('DemoButton', () => {
     it('renders the button', () => {
-        render(<DemoButton label="Charger la démo" handleClick={vi.fn()} />)
+        render(<DemoButton label="Load the demo" handleClick={vi.fn()} />)
         screen.getByRole('button', {
-            name: /Charger la démo/,
+            name: /Load the demo/,
```

Comment:
```suggestion
            name: /Load demo/,
```

I usually see `Load demo` without `the` on some website. It may be better grammatically ?

Reply:
It's been corrected, thank you.